### PR TITLE
Add Documentation for lightly/models/modules

### PR DESCRIPTION
**Pull request by ayush22iitbhu**
#1687

I have updated the docstrings for the following files:

1. lightly/models/modules/ijepa_timm.py
2. lightly/models/modules/ijepa.py
3. lightly/models/modules/masked_autoencoder_timm.py
4. lightly/models/modules/masked_autoencoder.py
5. lightly/models/modules/masked_casual_vision_transformer.py

### CODE REVIEW

File: lightly/models/modules/ijepa_timm.py [hunk 1]
```diff
@@ -98,6 +99,20 @@ def forward(
...
     ) -> torch.Tensor:
+        """Forward pass of the IJEPAPredictorTIMM.
+
+        Args:
+            x:
+                Input tensor.
+            masks_x:
+                Mask indices for the input tensor.
+            masks:
+                Mask indices for the predicted tokens.
```
Comment (MalteEbner) [inline lightly/models/modules/ijepa_timm.py:110]: Telling the shape of these tensors in the docstring would be nice.

**Review by MalteEbner — COMMENTED** [review]
This looks great, thanks a lot!

I made tow suggestions to make the docstrings a bit more helpful.

**Comment by ayush22iitbhu** [comment]
@MalteEbner I have done the changes for adding shape of tensors in docstring. Could you elaborate more on the adding typing in the function arguments?

**Comment by MalteEbner** [comment]
> @MalteEbner I have done the changes for adding shape of tensors in docstring. Could you elaborate more on the adding typing in the function arguments?

Hey @ayush22iitbhu,

thanks for adding the tensor shapes!
While testing if they are correct, I found out the the IJEPA module is broken anyway and created the corresponding Issue: @1720.
I reverted your last commit, as I would rather not document the shape than document a shape without testing if it is the correct shape. 

Sorry for this back-and-forth. I approved your commit as it is :)

**Review by MalteEbner — APPROVED** [review]
Well done!

**Pull request merged**